src/connections.jl

```diff
@@ -771,7 +770,7 @@ function Base.show(io::IO, jl_conn::Connection)
             print(io, "\n  ", ci_opt.keyword, " = ")
 
             if ci_opt.disptype == Password
-                print(io, "*" ^ ci_opt.dispsize)
+                print(io, "*"^ci_opt.dispsize)
```

Comment:
Do we want to maintain the lack of space around this operator when used with strings?

Reply:
i think yes